Modules/Core/Common/src/itkObjectFactoryBase.cxx

```diff
@@ -262,7 +262,7 @@ ObjectFactoryBase::Initialize()
     m_PimplGlobals->m_Initialized = true;
     ObjectFactoryBase::InitializeFactoryList();
     ObjectFactoryBase::RegisterInternal();
-#ifdef ITK_DYNAMIC_LOADING
+#if !defined(ITK_DYNAMIC_LOADING) && !defined(ITK_WRAPPING)
```

Comment:
```suggestion
#if defined(ITK_DYNAMIC_LOADING) && !defined(ITK_WRAPPING)
```

This would match the following table

| `ITK_DYNAMIC_LOADING` | `ITK_WRAPPING`     | Call `LoadDynamicFactories` | 
|-----------------------|--------------------|-----------------------------|
| :x:                   | :x:                |  :no_entry:         |
| :x:                   | :heavy_check_mark: |  :no_entry:         |
| :heavy_check_mark:    | :x:                |  :ok:                |
| :heavy_check_mark:    | :heavy_check_mark: |  :no_entry:         |

Reply:
+1

Reply:
Yes!